websites/T/Teams/dist/metadata.json

```diff
@@ -0,0 +1,161 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "MatStef132",
+    "id": "269787765120565248"
+  },
+  "url": "teams.microsoft.com",
+  "description": {
+    "en": "Stay connected and organized. Accomplish more together across work, school, and life with Microsoft Teams.",
+    "de": "Bleiben Sie gut vernetzt und organisiert – und erreichen Sie gemeinsam mehr mit Microsoft Teams: am Arbeitsplatz, in Schule und Studium und privat.",
+    "ru": "Будьте на связи и планируйте свои действия. Решайте поставленные задачи сообща и успевайте больше на работе, в школе и дома благодаря Microsoft Teams.",
+    "pl": "Bądź w kontakcie z innymi i zachowaj dobrą organizację pracy. Osiągaj więcej razem z innymi — w pracy, szkole i życiu — dzięki usłudze Microsoft Teams."
+  },
+  "service": "Microsoft Teams",
+  "version": "1.0",
```

Comment:
1.0.0

Reply:
Also to the messages I wanted an easy way for everyone to translate or change the text of the rich presence to how they want it.